packages/variables-plugin/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@utlitiywarehouse/variables-plugin",
```

Comment:
```suggestion
  "name": "@utilitywarehouse/variables-plugin",
```

Reply:
Could we also change the name to `figma-variables-plugin`? just to be more explicit.